wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -1628,7 +1628,7 @@ class SwiftGenerator private constructor(
     val enumName = type.typeName
     return TypeSpec.enumBuilder(enumName)
       .addModifiers(PUBLIC)
-      .addSuperType(UINT32)
+      .addSuperType(INT32)
```

Comment:
@JakeWharton This was introduced as UINT32 3 years ago with initial Swift support. Would it be possible to take a look at this change?

Reply:
No idea why. The docs mention it's signed (two's compliment) in multiple places.

Reply:
thank you!